package.json

```diff
@@ -83,6 +87,7 @@
     "globals": "^15.9.0",
     "mocha": "^10.7.3",
     "pino-colada": "^2.2.2",
+    "playwright-ctrf-json-reporter": "^0.0.17",
```

Comment:
Is this actually a devDep?

Reply:
said on the repo itself to save it as such https://github.com/ctrf-io/playwright-ctrf-json-report?tab=readme-ov-file#installation